crypto/http/http_client.c

```diff
@@ -937,6 +937,7 @@ OSSL_HTTP_REQ_CTX *OSSL_HTTP_open(const char *server, const char *port,
     }
     /* now overall_timeout is guaranteed to be >= 0 */
 
+    /* TODO adapt in order to fix callback design flaw, see #176088 */
```

Comment:
The number is wrong here. And remove TODO, the issue reference should be sufficient to find it.

Reply:
Oops, fixed.
And removed the mentioned TODO, here and in `apps.c`.